tofu/geom/_sampling_tools.pyx

```diff
@@ -1595,18 +1594,19 @@ cdef inline int  vmesh_disc_phi(int sz_r, int sz_z,
                 else:
                     ncells_rphi0 += <long>Cceil(twopi_over_dphi * disc_r0[jj])
                     tot_nc_plane[ii] = ncells_rphi0 * ncells_z
+
             # Get indices of phi
             # Get the extreme indices of the mesh elements that really need to
             # be created within those limits
             margin_step = margin * step_rphi[ii]
-            if abs0 - step_rphi[ii]*Cfloor(abs0 * inv_drphi) < margin_step:
-                nphi0 = int(Cround(min_phi_pi * inv_drphi))
+            if abs0 - step_rphi[ii]*Cfloor(abs0 / step_rphi[ii]) < margin_step:
+                nphi0 = int(Cround(min_phi_pi / step_rphi[ii]))
```

Comment:
Isn't it the same by definition of inv_drphi ?

Reply:
I thought so, but no. I found this example:
`abs0 / drphi` is  equal to 4, whereas `abs0 * inv_drphi` is equal to 3.999999...
When doing "floor" of the first one we found 4, and the second one 3... Which made the if statement False and everything else different